Makefile

```diff
@@ -1,3 +1,20 @@
+help:
+	@echo "Usage: make <target>"
```

Comment:
does mac have `make` autocomplete?  I know that on my linux machine I can tab autocomplete in this repo and it will give me all of the make options.  


ahhhh.  I see from https://www.gnu.org/software/make/manual/html_node/How-Make-Works.html  that `make` without arguments always performs the first recipe.  So, this is a nice add.